mbuild/compound.py

```diff
@@ -3228,6 +3242,58 @@ def from_parmed(self, structure, coords_only=False, infer_hierarchy=True):
             infer_hierarchy=infer_hierarchy,
         )
 
+    def to_rdkit(self):
```

Comment:
Missing docstring. Also, I think we should move the main `to_rdkit` code to `conversion.py` and just called that method here like we are doing with the other conversion methods.

Reply:
I think it is probably reasonable to move the bulk of the code to conversion.  We can just put a wrapper into compound (and probably do a PR moving other similar functions to their own relevant files.  I think for sure energy minimization probably should be put in an energy_minimization.py file or something).   I'll try to get the comments addressed and this finished up.

Reply:
I moved the bulk (pretty much all of the code) of `to_rdkit` over to `conversion` and call the method in compund's `to_rdkit`